src/interface/report/FakePlayerHelper.ts

```diff
@@ -0,0 +1,155 @@
+import SPECS from 'game/SPECS';
+
+export function fakePlayerInfoGenerator(player: any) {
+  const fakedPlayer = player;
+  fakedPlayer.gear = fakeGearGenerator(player.specID);
+  fakedPlayer.auras = fakeBuffGenerator();
+  fakedPlayer.error = null;
+  return fakedPlayer;
+}
+
+//region Gear Generation
+const CLOTH_SPECS = [SPECS.ARCANE_MAGE.id, SPECS.FIRE_MAGE.id, SPECS.FROST_MAGE.id, SPECS.DISCIPLINE_PRIEST.id, SPECS.HOLY_PRIEST.id, SPECS.SHADOW_PRIEST.id, SPECS.AFFLICTION_WARLOCK.id, SPECS.DEMONOLOGY_WARLOCK.id, SPECS.DESTRUCTION_WARLOCK.id];
+const LEATHER_SPECS = [SPECS.BALANCE_DRUID.id, SPECS.FERAL_DRUID.id, SPECS.GUARDIAN_DRUID.id, SPECS.RESTORATION_DRUID.id, SPECS.ASSASSINATION_ROGUE.id, SPECS.OUTLAW_ROGUE.id, SPECS.SUBTLETY_ROGUE.id, SPECS.BREWMASTER_MONK.id, SPECS.WINDWALKER_MONK.id, SPECS.MISTWEAVER_MONK.id, SPECS.HAVOC_DEMON_HUNTER.id, SPECS.VENGEANCE_DEMON_HUNTER.id];
+const MAIL_SPECS = [SPECS.BEAST_MASTERY_HUNTER.id, SPECS.MARKSMANSHIP_HUNTER.id, SPECS.SURVIVAL_HUNTER.id, SPECS.ELEMENTAL_SHAMAN.id, SPECS.ENHANCEMENT_SHAMAN.id, SPECS.RESTORATION_SHAMAN.id];
+const PLATE_SPECS = [SPECS.HOLY_PALADIN.id, SPECS.PROTECTION_PALADIN.id, SPECS.RETRIBUTION_PALADIN.id, SPECS.ARMS_WARRIOR.id, SPECS.FURY_WARRIOR.id, SPECS.PROTECTION_WARRIOR.id, SPECS.BLOOD_DEATH_KNIGHT.id, SPECS.FROST_DEATH_KNIGHT.id, SPECS.UNHOLY_DEATH_KNIGHT.id];
+
+const specificGearSets: { [key: number]: {} } = {
```

Comment:
What determines the value of they key?  Based the existing entries it looks to be something specific but I'm not sure what.

Reply:
We're still getting the specID from beta logs atm, so that's what I'm using. 
https://github.com/WoWAnalyzer/WoWAnalyzer/pull/3697/files#diff-02f8971e74d3f1a3ef8b2a9a2a2fb697R129 
here is where the function is called, and on line 5 in FakePlayerHelper you can see I call the function with "player.specID" parameter